### PR TITLE
Lint error on -> console log

### PR DESCRIPTION
**Pull request by timarney**
Add lint error for any console messages.

### CODE REVIEW

**Comment by timarney** [comment]
cc: @dsamojlenko 

Screen Shot 2020-07-29 at 2 55 23 PM

**Comment by jeberhardt** [comment]
This should also pop up a warning in IDEs such as WebStorm.

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,5 @@ ios/assets
 .env.release
 
 artifacts/*
+
+coverage/*
```
Comment (jeberhardt) [inline .gitignore:106]: Technically, not a part of this PR.

Comment (timarney) [reply .gitignore]: removed

File: .eslintrc.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ module.exports = {
     'import/no-named-as-default': 'off',
     '@shopify/images-no-direct-imports': 'off',
     '@shopify/jest/no-snapshots': 'off',
-    'no-console': 'off',
+    'no-console': 2,
```
Comment (jeberhardt) [inline .eslintrc.js:22]: Perhaps using `error` would be more clear as to what this is doing?

Comment (timarney) [reply .eslintrc.js]: using error now - good call

**Pull request merged**